.editorconfig

```
@@ -209,9 +209,7 @@ indent_size = 4
 max_line_length = 140
 ij_java_class_count_to_use_import_on_demand = 999
 ij_java_names_count_to_use_import_on_demand = 999
-# The first '@*,' is a workaround for https://youtrack.jetbrains.com/issue/IDEA-368382/Auto-import-puts-java-imports-first-even-when-Editor-Code-style-Java-Import-layout-puts-them-last
-# it should be removed once that is fixed
-ij_java_imports_layout = @*,*,|,com.**,|,org.**,|,java.**,|,javax.**,|,$*
+ij_java_imports_layout = *,|,com.**,|,io.**,|,org.**,|,java.**,|,javax.**,|,$*
```

```suggestion
ij_java_imports_layout = *,|,com.**,|,org.**,|,java.**,|,javax.**,|,$*
```

Let's revert my previous change as well.